src/main/kotlin/uk/gov/justice/hmpps/offendersearch/dto/MatchRequest.kt

```diff
@@ -8,11 +8,11 @@ import javax.validation.constraints.Past
 data class MatchRequest(
     @ApiModelProperty(value = "Offender first name", example = "john",  position = 1) val firstName: String? = null,
     @field:NotBlank(message = "Surname is required")
-    @ApiModelProperty(value = "Offender surname", example = "smith", position = 2) val surname: String? = null,
+    @ApiModelProperty(value = "Offender surname", example = "smith", position = 2) val surname: String?,
```

Comment:
remove defaults for required

Reply:
:+1: , but does the field need to be nullable?

Reply:
Yeah, I'm afraid so,  because when the client incorrectly passes a null you get a null exception rather than a validation exception - not sure if this is a workaround, almost like you want a lateinit type thing.